packages/expo/src/logs/LogSerialization.ts

```diff
@@ -73,7 +73,17 @@ function _stringifyLogData(data: unknown[]): string[] {
     if (typeof item === 'string') {
       return item;
     } else {
-      return prettyFormat(item, { plugins: [ReactNodeFormatter] });
+      //check the size of string returned
```

Comment:
style: space between `//` and comment

```suggestion
      // check the size of string returned
```